### PR TITLE
added readme.md to developing.md #issue 1033

### PR DESCRIPTION
**Pull request by Anash3**

### CODE REVIEW

File: docs/development/developing.md [hunk 1]
```diff
@@ -1,3 +1,56 @@
+# PyScript
+
+## What is PyScript
```
Comment (FabioRosado) [inline docs/development/developing.md:3]: I don't think this should be the first thing here.

This page is related to developing, using pyscript as a header will override the development title below (pages should only have one h1 header).

File: docs/development/developing.md [hunk 2]
```diff
@@ -1,3 +1,56 @@
...
+## Try PyScript
+
+To try PyScript, import the appropriate pyscript files into the ```<head>``` tag of your html page with:
+```html
+<head>
+<link rel="stylesheet" href="https://pyscript.net/latest/pyscript.css" />
+<script defer src="https://pyscript.net/latest/pyscript.js"></script>
+</head>
+```
+You can then use PyScript components in your html page. PyScript currently implements the following elements:
```
Comment (FabioRosado) [inline docs/development/developing.md:25]: There are more elements for example `<py-terminal>`

File: docs/development/developing.md [hunk 3]
```diff
@@ -1,3 +1,56 @@
...
+
+* `<py-script>`: can be used to define python code that is executable within the web page. The element itself is not rendered to the page and is only used to add logic
+* `<py-repl>`: creates a REPL component that is rendered to the page as a code editor and allows users to write executable code
+
+Check out the [the examples directory](examples) folder for more examples on how to use it, all you need to do is open them in Chrome.
```
Comment (FabioRosado) [inline docs/development/developing.md:30]: This examples link will be broken since docs don't know about the examples folder 😃

**Review by FabioRosado — CHANGES_REQUESTED** [review]
Hello @Anash3  thank you so much for working on this and creating this PR!

I think we want to avoid copying the readme contents into developing since they serve two different purposes. 

Ideally, what we should do is add a link to the developing page in the readme. When #1033 was opened, the README had a bunch of information that was moved to this developing page 😄

File: docs/development/developing.md (hunk 1 above)
Comment (Anash3) [reply docs/development/developing.md:3]: oh ok

Comment (Anash3) [reply docs/development/developing.md:3]: where should i add link to developing page in readme

Comment (FabioRosado) [reply docs/development/developing.md:3]: I think it should be under the `How to Contribute` in the Readme 😄

**Comment by FabioRosado** [comment]
I just thought that another thing we could do is copying the `Resources` links into developing 😄

**Pull request closed without merging**